Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -365,39 +372,148 @@ def start_chrome_headless(chrome_port, chrome_binary=CHROME_EXE, user_options=""
     return None, None
 
 
-def terminate_chrome(browser):
-    global CHROME_PROCESS
-    demisto.debug(f'terminate_chrome, {CHROME_PROCESS=}')
+def terminate_chrome(chrome_port=None, killall=False):
+    processes = subprocess.check_output(['ps', 'auxww'], stderr=subprocess.STDOUT, text=True).splitlines()
+    chrome_renderer_identifiers = ["--type=renderer"]
+    chrome_identifiers = ["chrom", "headless"] if killall else ["chrom", "headless", f"--remote-debugging-port={chrome_port}"]
+    process_in_list = [process for process in processes
+                       if all(identifier in process for identifier in chrome_identifiers)
+                       and not any(identifier in process for identifier in chrome_renderer_identifiers)]
 
-    threading.excepthook = excepthook_recv_loop
+    if killall:
+        pids = [int(process.split()[1]) for process in process_in_list]
+    else:
+        process_string_representation = process_in_list[0]
+        pids = [int(process_string_representation.split()[1])]
 
-    if CHROME_PROCESS:
-        demisto.debug(f'terminate_chrome, {CHROME_PROCESS=}')
-        CHROME_PROCESS.kill()
-        CHROME_PROCESS = None
+    for pid in pids:
+        process = psutil.Process(pid)
+        if process:
+            demisto.debug(f'terminate_chrome, {process=} on port {chrome_port}')
+            process.kill()
```

Comment:
Surround with `try-except`. Something like this:
```suggestion
            try:
                process.kill()
            except Exception as e:
                demisto.info(f"Exception when trying to kiil chrome with {pid=}, {e}")
```